src/modals/import-account-modal/import-account-type/import-account-type.tsx

```diff
@@ -0,0 +1,61 @@
+import { Formik } from 'formik';
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import { View } from 'react-native';
+
+import { ButtonLargePrimary } from '../../../components/button/button-large/button-large-primary/button-large-primary';
+import { ButtonLargeSecondary } from '../../../components/button/button-large/button-large-secondary/button-large-secondary';
+import { ButtonsContainer } from '../../../components/button/buttons-container/buttons-container';
+import { Divider } from '../../../components/divider/divider';
+import { InsetSubstitute } from '../../../components/inset-substitute/inset-substitute';
+import { Label } from '../../../components/label/label';
+import { ScreenContainer } from '../../../components/screen-container/screen-container';
+import { RadioButton } from '../../../components/styled-radio-buttons-group/styled-radio-buttons-group';
+import { FormRadioButtonsGroup } from '../../../form/form-radio-buttons-group';
+import { ImportAccountTypeEnum, ImportAccountTypeValues } from '../../../interfaces/import-account-type';
+import { useNavigation } from '../../../navigator/hooks/use-navigation.hook';
+import { formatSize } from '../../../styles/format-size';
+import { importAccountTypeInitialValues, importAccountTypeValidationSchema } from '../import-account-modal.form';
+
+interface Props {
+  importAccountStep: number;
+  setImportAccountStep: Dispatch<SetStateAction<number>>;
+  setImportType: Dispatch<SetStateAction<ImportAccountTypeEnum>>;
+}
+
+export const ImportAccountType: FC<Props> = ({ importAccountStep, setImportAccountStep, setImportType }) => {
+  const { goBack } = useNavigation();
+  const typeRadioButtons: RadioButton<ImportAccountTypeEnum>[] = [
+    { value: ImportAccountTypeEnum.PRIVATE_KEY, label: 'Private key' },
+    { value: ImportAccountTypeEnum.SEED_PHRASE, label: 'Seed phrase' }
+  ];
```

Comment:
move this variable out of component (it have no deps) to not let it reinitialise on each rerender